cmd/mattermost/commands/team_test.go

```diff
@@ -95,3 +95,20 @@ func TestListTeams(t *testing.T) {
 		t.Fatal("should have the created team")
 	}
 }
+
+func TestSearchTeams(t *testing.T) {
+	th := api4.Setup().InitBasic()
+	defer th.TearDown()
+
+	id := model.NewId()
+	name := "name" + id
+	displayName := "Name " + id
+
+	CheckCommand(t, "team", "create", "--name", name, "--display_name", displayName)
+
+	output := CheckCommand(t, "team", "search", name)
+
+	if !strings.Contains(string(output), name) {
+		t.Fatal("should have the created team")
+	}
+}
```

Comment:
Wouldn't mind seeing some more cases covered in the unit test

Reply:
As the `SearchAll` method searches teams by `Name` and `DisplayName`, I'm gonna add a test case to test searching of teams by display names.